app/code/Magento/Theme/Model/Theme.php

```diff
@@ -273,7 +275,18 @@ public function getArea()
     }
```

Comment:
can we refactor this method to simply ? Just call directly return getData('area'), Remove redundance code above

CC: @ihor-sviziev

Reply:
Just curious should use Magento\Framework\App\Area::PARAM_AREA instead string `area`

Reply:
Just curious should use Magento\Framework\App\Area::PARAM_AREA instead string area